components/Footer.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import Link from 'next/link'
+import { FaFacebook, FaInstagram, FaLinkedin, FaTwitter } from 'react-icons/fa'
+
+import { MdEmail } from 'react-icons/md'
+
+const SocialLinks = [
+  {
+    path: 'https://www.facebook.com/cutiehack/',
+    title: 'Facebook',
+    image: <FaFacebook size="56"></FaFacebook>,
```

Comment:
i dont think we need the size tbh. it looks pretty big rn and i think we can just use whatever the default size (without specifying size)

Reply:
i removed the size, but imma add it back cuz it looks like this without it
![image](https://user-images.githubusercontent.com/74090811/188061036-02a0de9d-169c-42c5-b8c7-89e2643d7332.png)

Reply:
ah oki oki